packages/gatsby-plugin-treat/tests/styling.test.js

```diff
@@ -24,22 +27,26 @@ async function navigateToServerWhenReady() {
 }
 
 async function startDevServer() {
+  console.log('startDevServer');
   gatsbyProcess = execa(`${gatsbyBinaryPath} develop -p ${gatsbyServerPort}`, gatsbyExecArgs);
-  // gatsbyProcess.stdout.pipe(process.stdout);
-  // gatsbyProcess.stderr.pipe(process.stdout);
+  gatsbyProcess.stdout.pipe(process.stdout);
+  gatsbyProcess.stderr.pipe(process.stdout);
   await navigateToServerWhenReady();
 }
 
 async function startProdServer() {
+  console.log('startProdServer');
   await execa(`${gatsbyBinaryPath} build`, gatsbyExecArgs);
   gatsbyProcess = execa(`${gatsbyBinaryPath} serve -p ${gatsbyServerPort}`, {shell: true, cwd: gatsbyFixturePath});
-  // gatsbyProcess.stdout.pipe(process.stdout);
-  // gatsbyProcess.stderr.pipe(process.stdout);
+  gatsbyProcess.stdout.pipe(process.stdout);
+  gatsbyProcess.stderr.pipe(process.stdout);
   await navigateToServerWhenReady();
 }
 
 async function stopServer() {
+  console.log('stopServer')
   await Promise.race(gatsbyProcess, gatsbyProcess.kill());
```

Comment:
Not sure `Promise.race` is correct here as neither functions return promises.
Does the following work?

```js
gatsbyProcess.kill({ forceKillAfterTimeout: 1000 })
```

Reply:
Thanks! Yeah, I took a quick look at `kill.js` and saw some usage of promises and hoped for the best, but you're correct, looking more closely it looks like it returns the result of `child_process.kill()` which is a boolean.